doc/src/angle_lepton.rst

```diff
@@ -41,6 +51,13 @@ angle coefficient.  For example `"200.0*theta^2"` represents a
 
    U_{angle,i} = K (\theta_i - \theta_0)^2 = K \theta^2 \qquad \theta = \theta_i - \theta_0
 
+.. versionchanged:: TBD
+
+By default the potential energy U is shifted so that he value U is 0.0
```

Comment:
Typo: the value U

Reply:
Also, should this be lepton/sphere?
https://github.com/lammps/lammps/blob/2d8f622c6af9aa3ab5733fdfb44e852898631e8d/doc/src/pair_lepton.rst?plain=1#L75-L76

Reply:
Thanks for spotting these.